package.json

```diff
@@ -82,7 +82,6 @@
     "react-intl": "^5.4.5",
     "react-native": "0.66.4",
     "react-native-android-open-settings": "^1.3.0",
-    "react-native-build-config": "^0.3.2",
```

Comment:
Was this dependency just not being used?

Reply:
Yeah that's what it looked like. It publishes to jcenter so figured just worth removing to avoid unnecessary potential issues (although I don't think it caused any since it wasn't being used)

although, i'm starting to wonder if maybe i had asked about this before. vaguely remember @gmaclennan saying it was being used indirectly but could be misremembering.

Reply:
Oh, nevermind I see that it wasn't being used in your PR description!